data/ships.txt

```diff
@@ -688,18 +688,18 @@ ship "Carrier"
 		
 	engine -35 200
 	engine 35 200
-	gun -25 -166 "Particle Cannon"
-	gun 25 -166 "Particle Cannon"
 	gun -16 -176 "Particle Cannon"
 	gun 16 -176 "Particle Cannon"
-	gun -25 -166 "Meteor Missile Launcher"
-	gun 25 -166 "Meteor Missile Launcher"
 	gun -16 -176 "Meteor Missile Launcher"
 	gun 16 -176 "Meteor Missile Launcher"
-	turret 0 15 "Heavy Laser Turret"
+	gun -25 -166 "Particle Cannon"
+	gun 25 -166 "Particle Cannon"
+	gun -25 -166 "Meteor Missile Launcher"
+	gun 25 -166 "Meteor Missile Launcher"
 	turret 0 -114 "Heavy Laser Turret"
-	turret 0 157 "Heavy Laser Turret"
-	turret 0 103 "Heavy Anti-Missile Turret"
+	turret 0 15 "Heavy Laser Turret"
+	turret 0 103 "Heavy Laser Turret"
+	turret 0 157 "Heavy Anti-Missile Turret"
```

Comment:
Is this right?
In this version, the anti-missile is in the far back. But in the original, it was in between the second and third heavy laser turrets.